pool/paymentmgr.go

```diff
@@ -395,51 +394,12 @@ func (pm *PaymentMgr) sharePercentages(shares []*Share) (map[string]*big.Rat, er
 	return percentages, nil
 }
 
-// PPSEligibleShares fetches all shares created before or at the provided
-// time.
-func (pm *PaymentMgr) PPSEligibleShares(max []byte) ([]*Share, error) {
```

Comment:
Let's give the `paymentmgr` as `chainstate` the same treatment: maintain the current api and extract unexported db helpers where needed.

Reply:
Done

Reply:
On further discussion we've agreed to make the needed api changes since the functions that are currently being made standalone will end up as part of a db driver in the long run and also eliminate the amount or wrapping needed to maintain the current api. Do proceed.